.github/workflows/query-engine-driver-adapters.yml

```diff
@@ -61,14 +61,6 @@ jobs:
         run: |
           echo "STORE_PATH=$(pnpm store path --silent)" >> $GITHUB_ENV
 
-      - uses: actions/cache@v3
-        name: "Setup pnpm cache"
-        with:
-          path: ${{ env.STORE_PATH }}
-          key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
-          restore-keys: |
-            ${{ runner.os }}-pnpm-store-
-
```

Comment:
Any particular reason or context for this change?

Reply:
I was seeing weird errors yesterday that might have been caused by caching. I noticed the runtime of tests didn't grow significantly and the problems ceased (coincidentally) when I removed this block.